### DISCUSSION TITLE
Show tourist routes, hikes, cycle paths, etc.

### DISCUSSION THREAD

**Discussion by nonobio** [organicmaps/organicmaps#8233] [Ideas]
Hello,

I love Organic Maps, especially for saving bookmarks, thanks a lor for this app :)
But to find hikes or cycle paths, I use Mapy, which displays them in red, yellow, blue, pink according to categories on its "touristic" map:
![image](https://github.com/organicmaps/organicmaps/assets/7425799/c136124a-03ca-4f75-9e75-041c5909ae62)

Unless I'm mistaken, it is not possible to have these elements with Organic Maps:
![image](https://github.com/organicmaps/organicmaps/assets/7425799/992e4517-3121-460d-a997-40c752ccd092)

It would be great if they were there.
If that's not possible, too bad, I'll continue to use both apps :)

**Comment by biodranik** [comment]
Did you check #865 and other related issues/discussions?
